CMakeLists.txt

```
@@ -292,3 +296,12 @@ set(CPACK_RESOURCE_FILE_LICENSE "${Ginkgo_SOURCE_DIR}/LICENSE")
 set(CPACK_PACKAGE_ICON "${Ginkgo_SOURCE_DIR}/assets/logo.png")
 set(CPACK_PACKAGE_CONTACT "ginkgo.library@gmail.com")
 include(CPack)
+
+# And finally, print the configuration:
+#
+if(GINKGO_CONFIG_LOG_VERBOSITY MATCHES "1")
```

using BOOL would simplify this to
```suggestion
if(GINKGO_CONFIG_LOG_VERBOSITY)
```